### ISSUE TITLE
Automatic Equation Numbering

### ISSUE THREAD

**Issue by taxminion** [alshedivat/al-folio#158]
**Is your feature request related to a problem? Please describe.**
To have equation numbers I need to put \tag{[#]} and manually update/rearrange every equation. One of my blog posts has over 60 equations and the inability to rearrange any of it without manually updating up to 60 numbers is absolutely atrocious.

**Describe the solution you'd like**
Have an option in _config.yml to automatically label equations.
ex. have
```
$$\begin{equation}
e^{ix}=\cos(x)+i\sin(x)
\end{equation}$$
```
do the same thing as
```
$$\begin{equation} \tag{1}
e^{ix}=\cos(x)+i\sin(x)
\end{equation}$$
```
(assuming it's the first equation)

**Describe alternatives you've considered**
The ability to insert a script or bit of code as described [here](http://docs.mathjax.org/en/latest/input/tex/eqnumbers.html) to be able to do this myself. This is probably possible, I just don't know that much js or html.

**Additional context**
To reference equations with `\label{[eqname]}` and` \ref{[eqname}}` the equation needs both `\tag{[#]}` and` \label{[eqname]} `

**Comment by alshedivat** [comment]
Added automatic equation numbering in #160. Check out the demo [here](https://alshedivat.github.io/al-folio/blog/2015/math/).
